Misc/NEWS.d/next/Library/2020-08-10-15-06-55.bpo-21041.cYz1eL.rst

```diff
@@ -0,0 +1 @@
+Pathlib path.parents now support negative indexing.
```

Comment:
```suggestion
Pathlib path.parents now support negative indexing. Patch contributed by Yaroslav Pankovych.
```

You can also add your name to https://github.com/python/cpython/blob/master/Misc/ACKS